src/overlays/actors/ovl_En_Ko/z_en_ko.c

```diff
@@ -1,21 +1,44 @@
+/*
+ * File: z_en_ko.c
+ * Overlay: ovl_En_Ko
+ * Description: Kokiri children, and Fado
+ */
+
 #include "z_en_ko.h"
+#include "objects/object_fa/object_fa.h"
+#include "vt.h"
 
 #define FLAGS 0x00000019
 
 #define THIS ((EnKo*)thisx)
+#define ENKO_TYPE (this->actor.params & 0xFF)
+#define ENKO_PATH ((this->actor.params & 0xFF00) >> 8)
 
 void EnKo_Init(Actor* thisx, GlobalContext* globalCtx);
 void EnKo_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnKo_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnKo_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-extern UNK_TYPE D_06006A60;
-extern UNK_TYPE D_06007830;
-extern UNK_TYPE D_06007D94;
-extern UNK_TYPE D_0600879C;
-extern UNK_TYPE D_06008F6C;
+void func_80A99048(EnKo* this, GlobalContext* globalCtx);
+void func_80A995CC(EnKo* this, GlobalContext* globalCtx);
+void func_80A99384(EnKo* this, GlobalContext* globalCtx);
+void func_80A99438(EnKo* this, GlobalContext* globalCtx);
+void func_80A99504(EnKo* this, GlobalContext* globalCtx);
+void func_80A99560(EnKo* this, GlobalContext* globalCtx);
+Gfx* EnKo_SetEnvColor(GraphicsContext* gfxCtx, u8 r, u8 g, u8 b, u8 a);
+s32 EnKo_OverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, void* thisx,
+                          Gfx** arg6);
+u16 func_80A97610(GlobalContext*, Actor*);
+s16 func_80A97738(GlobalContext*, Actor*);
+f32 func_80A97BC0(EnKo* this);
```

Comment:
Most of these forward declarations look like they aren't necessary, so we would normally omit them